webextensions/manifest/optional_host_permissions.json

```
@@ -0,0 +1,29 @@
+{
+  "webextensions": {
+    "manifest": {
+      "optional_host_permissions": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_host_permissions",
+          "support": {
+            "chrome": {
+              "version_added": "88",
```

I'm not sure where this is from, but the timing doesn't look right.  Chrome 88 was released in 2020, but we only discussed this in WECG in late 2021, and the chromium CL seems to have landed in 2022, so either Chrome 101 or 102.

https://github.com/w3c/webextensions/issues/119
https://issues.chromium.org/issues/40801157#comment9

---

@dotproto, can you confirm what the correct version number should be here? When I went looking for references to `optional_host_permissions,` following the "blame" seemed to lead me back to versions well before the official release of manifest V3.

---

I can confirm that `"optional_host_permissions"` was introduced after initial MV3 support landed in Chrome. Per the Chrome's [Whats new in Chrome extensions](https://developer.chrome.com/docs/extensions/whats-new#m102-optional-host-permissions) page, support for this feature landed in 102. 

```suggestion
              "version_added": "102",
```